test/dotnet-pack.Tests/PackTests.cs

```diff
@@ -95,6 +95,29 @@ public void HasBuildOutputWhenUsingBuildBasePath()
             zip.Entries.Should().Contain(e => e.FullName == "lib/netstandardapp1.5/TestLibraryWithConfiguration.dll");
         }
 
+        [Fact]
+        public void PackAddsCorrectFilesForProjectsWithOutputNameSpecified()
```

Comment:
I don't see the equivalent test for the default Project.Name.

Now that we're adding this way to switch between them I think it'd be valuable to have both tests.